lib/at_host/at_host.c

```diff
@@ -170,6 +196,15 @@ static void uart_rx_handler(uint8_t character)
 	}
 }
 
+void at_cmd_set_custom_handler(at_cmd_custom_handler_t handler)
```

Comment:
Maybe instead of using a fixed prefix configured by Kconfig, the application could register the prefix along with the handler? This would also leave an opening to extend the implementation in the future, if multiple handlers for different prefixes were needed.